driver/esp_camera.c

```diff
@@ -156,9 +157,17 @@ static esp_err_t camera_probe(const camera_config_t *config, camera_model_t *out
         CAMERA_ENABLE_OUT_CLOCK(config);
     }
 
-    if (config->pin_sscb_sda != -1) {
-        ESP_LOGD(TAG, "Initializing SSCB");
-        SCCB_Init(config->pin_sscb_sda, config->pin_sscb_scl);
+    if (config->pin_sccb_sda != -1) {
```

Comment:
In the header file comment you mention `NO_PIN`, could we use the same macro here?

Reply:
This lib uses `-1` in most cases, so it be changed to `-1` here, and relevant comments have been corrected.